modules/openapi-generator/src/main/resources/typescript/auth/auth.mustache

```diff
@@ -0,0 +1,152 @@
+import {RequestContext} from '../http/http';
+// typings for btoa are incorrect
+//@ts-ignore
+import * as btoa from "btoa";
```

Comment:
is this necessary in the browser?

Reply:
The polyfill would be needed in IE ≤9 and Opera ≤ 10.1.
I would be fine with only including it in node context.
What do you think, @TiFu ?

Reply:
Agreed - I'll exclude it